tests/ops/service/storage_uploader_service_test.py

```diff
@@ -76,6 +78,52 @@ def test_uploader_s3_success(
         f"{request_id}.json",
         "json",
         request_id,
+        S3AuthMethod.SECRET_KEYS.value,
+    )
+
+    storage_config.delete(db)
+
+
+@mock.patch("fidesops.ops.service.storage.storage_uploader_service.upload_to_s3")
+def test_uploader_s3_success_automatic_auth(
+    mock_upload_to_s3: Mock, db: Session, privacy_request
+) -> None:
+    request_id = privacy_request.id
+
+    mock_config = {
+        "name": "test dest",
+        "key": "test_dest_key",
+        "type": StorageType.s3.value,
+        "details": {
+            "auth_method": S3AuthMethod.AUTOMATIC.value,
+            "bucket": "some-bucket",
+            "naming": FileNaming.request_id.value,
+            "max_retries": 10,
+            "object_name": "something",
+        },
+    }
+    storage_config = StorageConfig.create(db, data=mock_config)
+
+    mock_upload_to_s3.return_value = (
+        f"https://some-bucket.s3.amazonaws.com/{request_id}.json"
+    )
+    upload_data = {"phone": "1231231234"}
```

Comment:
This will probably never matter, but if we ever had a phone validator this isn't a valid phone number. Area code can't start with 0 or 1

```suggestion
    upload_data = {"phone": "2018675309"}
```